pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "3.9.*"
 disnake = "~=2.4"
 # See https://bot-core.pythondiscord.com/ for docs.
-bot-core = {url = "https://github.com/python-discord/bot-core/archive/511bcba1b0196cd498c707a525ea56921bd971db.zip"}
+bot-core = {url = "https://github.com/python-discord/bot-core/archive/5593a0c95e2b4fd9515c8875ea6e51c0ab88eb00.zip"}
```

Comment:
We basically do versioning already for the change log. Do we plan to use git tags? Then I think this URL could use the tag, which would be clearer than a hash.

Reply:
Yea, good idea. https://github.com/python-discord/bot/pull/2103/commits/3a03e83776db009494f189183189615d616fdb59